### PR TITLE
Implement the delete role

### PR DESCRIPTION
**Pull request by LinconOssa**
The implementation of Delete is done

### CODE REVIEW

File: src/pages/privileges/outlets/roles/add-role/interface.tsx [hunk 1]
```diff
@@ -130,126 +134,135 @@ export function AddRolUI(props: AddRolUIProps) {
...
-            <PageTitle
-              title={createRolConfig[0].title}
-              description={createRolConfig[0].description}
-              navigatePage="/privileges/roles"
-            />
+    <>
+      {loading ? (
+        <LoadingApp />
+      ) : (
+        <Stack direction="column" padding={smallScreen ? "s200" : "s400 s800"}>
```
Comment (ANGIECATHERINEP) [inline src/pages/privileges/outlets/roles/add-role/interface.tsx:141]: change the padding value to px

File: src/pages/privileges/outlets/roles/delete-role/index.tsx [hunk 2]
```diff
@@ -5,36 +5,61 @@ import { functionById } from "@src/mocks/utils/dataMock.service";
...
+  //   await handleDeleteRol({
+  //     key: "k_Rol",
+  //     nameDB: "linix-roles",
+  //     identifier: rol,
+  //   });
+  //   setShowModal(false);
+  //   setLoading(true);
+  //   setIdDeleted(rol);
+  // };
+
```
Comment (ANGIECATHERINEP) [inline src/pages/privileges/outlets/roles/delete-role/index.tsx:58]: delete these comments

File: src/pages/privileges/outlets/roles/index.tsx [hunk 3]
```diff
@@ -15,34 +17,61 @@ export function Roles() {
...
+  //   const filterRecordRemoved = linixRoles.filter(
+  //     (linixRol) => linixRol.k_Rol !== idDeleted
+  //   );
+  //   filterRecordRemoved &&
+  //     setMessage({
+  //       visible: true,
+  //       data: generalMessage.success,
+  //     });
+  //   // eslint-disable-next-line react-hooks/exhaustive-deps
+  // }, [idDeleted]);
```
Comment (ANGIECATHERINEP) [inline src/pages/privileges/outlets/roles/index.tsx:74]: delete these comments

File: src/services/roles/useCasesByRole/mappers.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+const mapUseCasesByRoleByRoleFormatsApiToEntity = (
+  reportsClientServerFormat: Record<string, string | number | object>
```
Comment (ANGIECATHERINEP) [inline src/services/roles/useCasesByRole/mappers.ts:2]: rename prop

File: src/services/roles/useCasesByRole/mappers.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    k_Rol: String(reportsClientServerFormat.k_Rol),
+    k_Usecase: String(reportsClientServerFormat.k_Usecase),
+    n_Usecase: String(reportsClientServerFormat.n_Usecase),
+    i_Privi: Boolean(reportsClientServerFormat.i_Privi),
+  };
+  return buildreportsClientServerFormat;
+};
+
+const mapUseCasesByRoleByRoleFormatsApiToEntities = (
+  reportsClientServerFormat: Record<string, string | number | object>[]
```
Comment (ANGIECATHERINEP) [inline src/services/roles/useCasesByRole/mappers.ts:14]: rename prop

**Review by ANGIECATHERINEP — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/roles/delete-role/index.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/roles/delete-role/index.tsx:58]: Resolved

File: src/pages/privileges/outlets/roles/index.tsx (hunk 3 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/roles/index.tsx:74]: Resolved

File: src/services/roles/useCasesByRole/mappers.ts (hunk 4 above)
Comment (LinconOssa) [reply src/services/roles/useCasesByRole/mappers.ts:2]: Resolved

File: src/services/roles/useCasesByRole/mappers.ts (hunk 5 above)
Comment (LinconOssa) [reply src/services/roles/useCasesByRole/mappers.ts:14]: Resolved

**Review by ANGIECATHERINEP — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**